src/lib/analytics/analytics.js

```diff
@@ -361,14 +365,31 @@ const patchLogger = () => {
   const debounceFireEvent = debounce(fireEvent, 500, { leading: true })
 
   logger.error = (...args) => {
-    const [logContext, logMessage, eMsg, errorObj, ...rest] = args
+    const [logContext, logMessage, eMsg, errorObj, _extra = {}] = args
+    let { dialogShown = true, category = ERROR_CATEGORY_UNEXPECTED, ...extra } = _extra
```

Comment:
set it false by default. this allows you do not pass dialogSowh: false everywhere except the few cases where it's neede only

```suggestion
    const { dialogShown = false, category = ERROR_CATEGORY_UNEXPECTED, ...extra } = _extra
```